src/cuerdas/core.cljc

```diff
@@ -676,3 +676,17 @@
             second
             (split suffix)
             first)))
+
+(defn <<
+  "Unindent multiline text.
+  Uses either a supplied regex or the shortest
+  beginning-of-line to non-whitespace distance"
+  ([r s] (->> s lines (map #(replace % r "")) unlines))
+  ([s]
+   (let [all-indents (->> (rest (lines s)) ;; ignore the first line
```

Comment:
Please, can you sort the arities in the ascendant order, `(fn ([x] ...) ([x y] ...) ([x y z] ...))`